nixpkgs: add packages

MarcelCoding opened the pull request.

MarcelCoding committed:
nixpkgs: add packages

MarcelCoding committed:
search: update to packages

MarcelCoding commented:
idk, could be

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'authentik-nix':
    'github:nix-community/authentik-nix/e929253deddb8398c7010b28187d9d43603db950?narHash=sha256-HdGpVTqPguAvkDbgjWi38sGUaI0MvxJJX2bFVZTKXaE%3D' (2025-12-17)
  → 'github:nix-community/authentik-nix/94c544f6cd51735728391c8e9463efc893ddf26b?narHash=sha256-jIYGoq90mDkeVEM9r9CHa/3H1ByYp7ZkE0IRf%2BhaysE%3D' (2026-01-12)
• Updated input 'comin':
    'github:nlewo/comin/b26f456f23aa3f515140cf74ca8eba2b45851e44?narHash=sha256-HzPyeo0B2wCMToYNUPfBA0Q4UB8mlgj7XgCRBlV9sDU%3D' (2026-01-10)
  → 'github:nlewo/comin/e6eb852b0ae463b515293fd77d62e798ba238683?narHash=sha256-w/jPLcBd67ztB9Utk8jgYrY56uHD0waaTG8KflOG7EM%3D' (2026-01-11)
• Updated input 'flake-parts':
    'github:hercules-ci/flake-parts/250481aafeb741edfe23d29195671c19b36b6dca?narHash=sha256-feveD98mQpptwrAEggBQKJTYbvwwglSbOv53uCfH9PY%3D' (2026-01-05)
  → 'github:hercules-ci/flake-parts/80daad04eddbbf5a4d883996a73f3f542fa437ac?narHash=sha256-PVvu7OqHBGWN16zSi6tEmPwwHQ4rLPU9Plvs8/1TUBY%3D' (2026-01-11)
• Updated input 'home-manager':
    'github:nix-community/home-manager/8bc5473b6bc2b6e1529a9c4040411e1199c43b4c?narHash=sha256-bAXnnJZKJiF7Xr6eNW6%2BPhBf1lg2P1aFUO9%2BxgWkXfA%3D' (2026-01-10)
  → 'github:nix-community/home-manager/b1fa714d6cd656e3105d1965637be6ab7541d7d7?narHash=sha256-mBKqOJkxCRwEhIXfq93WTcDXsBlJ/f1Dfv9thJxrDPs%3D' (2026-01-13)
• Updated input 'lanzaboote':
    'github:nix-community/lanzaboote/657469e8f036334db768daaf7732b1174676054b?narHash=sha256-0iVZ99H3kR5h6Lhw8kDDuUc5C/k6iismeWgCS1qWTQ4%3D' (2026-01-06)
  → 'github:nix-community/lanzaboote/7e031eb535a494582f4fc58735b5aecba7b57058?narHash=sha256-3yDvlAqWa0Vk3B9hFRJJrSs1xc%2BFwVQFLtu//VrTR4c%3D' (2026-01-13)
• Updated input 'lanzaboote/crane':
    'github:ipetkov/crane/7d59256814085fd9666a2ae3e774dc5ee216b630?narHash=sha256-TH/xTeq/RI%2BDOzo%2Bc%2B4F431eVuBpYVwQwBxzURe7kcI%3D' (2026-01-03)
  → 'github:ipetkov/crane/2fb033290bf6b23f226d4c8b32f7f7a16b043d7e?narHash=sha256-9/9ntI0D%2BHbN4G0TrK3KmHbTvwgswz7p8IEJsWyef8Q%3D' (2026-01-07)
• Updated input 'microvm':
    'github:microvm-nix/microvm.nix/ad2ef577357f378e3dc1c222851dbc1ede2b044c?narHash=sha256-JC%2Bu/QQ49q9%2B/zfsO06u6NhBy/6o6WH8ad3BtTAcoeo%3D' (2026-01-08)
  → 'github:microvm-nix/microvm.nix/4f267df275361406cda9a3f8e349035be207b307?narHash=sha256-VmLSHlimAq%2Bem9rXX9YhBS0Shu5MBCAQi2Kd//8OOgQ%3D' (2026-01-10)
• Updated input 'nix-darwin':
    'github:nix-darwin/nix-darwin/9f48ffaca1f44b3e590976b4da8666a9e86e6eb1?narHash=sha256-V%2BVkFs0aSG0ca8p/N3gib7FAf4cq9jyr5Gm%2BZBrHQpo%3D' (2026-01-06)
  → 'github:nix-darwin/nix-darwin/7b1d394e7d9112d4060e12ef3271b38a7c43e83b?narHash=sha256-8wMrJP/Xk5Dkm0TxzaERLt3eGFEhHTWaJKUpK3AoL4o%3D' (2026-01-12)
• Updated input 'rust-overlay':
    'github:oxalica/rust-overlay/312b4371e72f644ffcff25b23615195e3b390643?narHash=sha256-HFFVQaux1JoOjEvgBT0ASk1Je%2BjsipyO5c91FoLMed8%3D' (2026-01-10)
  → 'github:oxalica/rust-overlay/03dda130a8701b08b0347fcaf850a190c53a3c1e?narHash=sha256-Tg/kL8eKMpZtceDvBDQYU8zowgpr7ucFRnpP/AtfuRM%3D' (2026-01-13)
• Updated input 'sops-nix':
    'github:Mic92/sops-nix/a8cfe238b93166f9f96c0df67a94e572554ee624?narHash=sha256-BVpTd93G0XmAK1iXiBdhUA5Uvt%2BWmM1YL0mA4REcT68%3D' (2026-01-10)
  → 'github:Mic92/sops-nix/691b8b6713855d0fe463993867291c158472fc6f?narHash=sha256-jJqlW8A3OZ5tYbXphF7U8P8g/3Cn8PPwPa4YlJ/9agg%3D' (2026-01-13)

SuperSandro2000 committed:
lanzaboote pkgs

SuperSandro2000 commented on flake.nix:
```diff
@@ -249,13 +252,19 @@
+                    }
+                  '';
                 }
                 # nixos-apple-silicon
```
```suggestion
                # nixos-apple-silicon
                # tries to load files from /boot with --impure
```

SuperSandro2000 commented on flake.nix:
```diff
@@ -228,6 +228,9 @@
                   ];
                   name = "Lanzaboote";
                   urlPrefix = "https://github.com/nix-community/lanzaboote/blob/master/";
+                  pkgs = pkgs.writeText "pkgs.nix" /* nix */ ''
```
```suggestion
                  # requires --impure
                  pkgs = pkgs.writeText "pkgs.nix" /* nix */ ''
```

SuperSandro2000 committed:
Add comments

MarcelCoding committed:
lanzaboote: fix builtins.getFlake

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'authentik-nix':
    'github:nix-community/authentik-nix/94c544f6cd51735728391c8e9463efc893ddf26b?narHash=sha256-jIYGoq90mDkeVEM9r9CHa/3H1ByYp7ZkE0IRf%2BhaysE%3D' (2026-01-12)
  → 'github:nix-community/authentik-nix/1cab906a5cb342a4890ea9e4fe3993c6d438689b?narHash=sha256-9eiLAIUI3rsjqdY32%2BjQdKB%2B0VI6Jks0uf0s/UVMVJI%3D' (2026-01-24)
• Updated input 'authentik-nix/authentik-src':
    'github:goauthentik/authentik/0d617e4ad1eb9e4540ba5381e6ce06e971affc63?narHash=sha256-d0pPNE2T30COdFse0T15Mx8XW4BGg8hgPQvmW2dAV9s%3D' (2025-12-16)
  → 'github:ma27/authentik/72ad5fe320f2201fc2a37372d4c9cb46377a58e5?narHash=sha256-HDTbQB/sMhYh2b95dQwzF8OgrwLWdl4hVmx6wtDcgE8%3D' (2026-01-16)
• Updated input 'bird-nix':
    'github:NuschtOS/bird.nix/48e0bbf3c221444905f375a94f6a8802f0349c6b?narHash=sha256-%2Bd8xTY8agz72%2BANMplTwP2IL6yk2fsgDLS0Di0dKo1U%3D' (2025-10-01)
  → 'github:NuschtOS/bird.nix/add53b9595e2e910ba58d552f2c42004babbb056?narHash=sha256-XsHjComfnibSuQtV/mfOVpLijXElPgt6e/2fVL1NAbs%3D' (2026-01-17)
• Updated input 'comin':
    'github:nlewo/comin/e6eb852b0ae463b515293fd77d62e798ba238683?narHash=sha256-w/jPLcBd67ztB9Utk8jgYrY56uHD0waaTG8KflOG7EM%3D' (2026-01-11)
  → 'github:nlewo/comin/a35bc15dacef048d3dbe611c91acf7a4c274f24e?narHash=sha256-41Ts3IL/6zD%2BenoQh3/8FCucCJn3HBsI8FbdNDG2MbU%3D' (2026-01-22)
• Updated input 'disko':
    'github:nix-community/disko/916506443ecd0d0b4a0f4cf9d40a3c22ce39b378?narHash=sha256-P0kM%2B5o%2BDKnB6raXgFEk3azw8Wqg5FL6wyl9jD%2BG5a4%3D' (2025-12-19)
  → 'github:nix-community/disko/00395d188e3594a1507f214a2f15d4ce5c07cb28?narHash=sha256-GVJ0jKsyXLuBzRMXCDY6D5J8wVdwP1DuQmmvYL/Vw/Q%3D' (2026-01-20)
• Updated input 'home-manager':
    'github:nix-community/home-manager/b1fa714d6cd656e3105d1965637be6ab7541d7d7?narHash=sha256-mBKqOJkxCRwEhIXfq93WTcDXsBlJ/f1Dfv9thJxrDPs%3D' (2026-01-13)
  → 'github:nix-community/home-manager/2539eba97a6df237d75617c25cd2dbef92df3d5b?narHash=sha256-6Cwtvzrw79cOk1lCzN2aKSVrpgSOSQoYhyMmhXXZjTA%3D' (2026-01-24)
• Updated input 'impermanence':
    'github:nix-community/impermanence/82e5bc4508cab9e8d5a136626276eb5bbce5e9c5?narHash=sha256-iyrn9AcPZCoyxX4OT8eMkBsjG7SRUQXXS/V1JzxS7rA%3D' (2026-01-07)
  → 'github:nix-community/impermanence/69ecf31e8fddc9354a4b418f3a517445d486bb54?narHash=sha256-OyxsfXNcOkt06/kM%2B4bnuC8moDx%2Bt7Qr%2BRB0BBa83Ig%3D' (2026-01-20)
• Updated input 'lanzaboote':
    'github:nix-community/lanzaboote/7e031eb535a494582f4fc58735b5aecba7b57058?narHash=sha256-3yDvlAqWa0Vk3B9hFRJJrSs1xc%2BFwVQFLtu//VrTR4c%3D' (2026-01-13)
  → 'github:nix-community/lanzaboote/1bea6e953d06da77729edd0004291ced527bcb4a?narHash=sha256-xGlAdk2c1mVxOTMzzCYHDYuXaBMoH1BTr2nJOGkY/SQ%3D' (2026-01-23)
• Updated input 'microvm':
    'github:microvm-nix/microvm.nix/4f267df275361406cda9a3f8e349035be207b307?narHash=sha256-VmLSHlimAq%2Bem9rXX9YhBS0Shu5MBCAQi2Kd//8OOgQ%3D' (2026-01-10)
  → 'github:microvm-nix/microvm.nix/f469c1dfede623bbbf1ac605f6359316fd4002ef?narHash=sha256-mKrMf7eG9TM2AM3pTuhIiCGmZ/JwDegCQH3ThVqcTuc%3D' (2026-01-17)
• Updated input 'nix-darwin':
    'github:nix-darwin/nix-darwin/7b1d394e7d9112d4060e12ef3271b38a7c43e83b?narHash=sha256-8wMrJP/Xk5Dkm0TxzaERLt3eGFEhHTWaJKUpK3AoL4o%3D' (2026-01-12)
  → 'github:nix-darwin/nix-darwin/0fc4e7ac670a0ed874abacf73c4b072a6a58064b?narHash=sha256-5ulSDyOG1U%2B1sJhkJHYsUOWEsmtLl97O0NTVMvgIVyc%3D' (2026-01-18)
• Updated input 'nixos-apple-silicon':
    'github:tpwrules/nixos-apple-silicon/530aa73aa9a21a078ff861b84767ae1d469715fa?narHash=sha256-njwPfL5gM/KRGLJ29CbQ4nLBoEkNPMvEHBAAPOP8BZc%3D' (2026-01-05)
  → 'github:tpwrules/nixos-apple-silicon/003afbd7b4f90c83497906ba2e9099874c2e3a4f?narHash=sha256-OA%2BQrrf0FWqARjgJASGrPKMUw237G3gYG%2BMQn4upABU%3D' (2026-01-23)
• Updated input 'nixos-hardware':
    'github:nixos/nixos-hardware/40b1a28dce561bea34858287fbb23052c3ee63fe?narHash=sha256-ljDBUDpD1Cg5n3mJI81Hz5qeZAwCGxon4kQW3Ho3%2B6Q%3D' (2025-12-31)
  → 'github:nixos/nixos-hardware/9f7ba891ea5fc3ededd7804f1a23fafadbcb26ca?narHash=sha256-3ymIZ8s3%2Bhu7sDl/Y48o6bwMxorfKrmn97KuWiw1vjY%3D' (2026-01-22)
• Updated input 'nixos-modules':
    'github:NuschtOS/nixos-modules/db6f2a33500dadb81020b6e5d4281b4820d1b862?narHash=sha256-gbhR8%2BgNrhmYxKaNJpTjFivuibr3ZdlB5eU0a8yE36I%3D' (2025-12-22)
  → 'github:NuschtOS/nixos-modules/31108e0d75bd47ddfc217b58df598e78fe3bcd42?narHash=sha256-hmIo/e6mo40Y2v1DaH2yTtvB3lZ/zcf6gVNmgYhBgYc%3D' (2026-01-15)
• Updated input 'nixos-wsl':
    'github:nix-community/NixOS-WSL/be4af8042e7a61fa12fda58fe9a3b3babdefe17b?narHash=sha256-55V0AJ36V5Egh4kMhWtDh117eE3GOjwq5LhwxDn9eHg%3D' (2025-12-15)
  → 'github:nix-community/NixOS-WSL/38a5250e57f583662eac3b944830e4b9e169e965?narHash=sha256-RY9kJDXD6%2B2Td/59LkZ0PFSereCXHdBX9wIkbYjRKCY%3D' (2026-01-24)
• Updated input 'nixpkgs':
    'github:nuschtos/nuschtpkgs/167560714b5c24ff564d27cab5ec1a717971a5a1?narHash=sha256-movudj7psSLJlnmrhxGokSmPzAsJF85VakeGQk7loi4%3D' (2026-01-12)
  → 'github:nuschtos/nuschtpkgs/347928902d4292ce08651b97c1264edf7a342db1?narHash=sha256-VyRjKmCv3iCDjSkDygD0YT97IB0xRQJO2qlsC/B6QSw%3D' (2026-01-22)
• Updated input 'nixvim':
    'github:nix-community/nixvim/7eb8f36f085b85a2aeff929aff52d0f6aa14e000?narHash=sha256-AoSWS8%2BP%2B7hQ/jIdv0wBjgH1MvnerdWBFXO4GV3JoQs%3D' (2026-01-08)
  → 'github:nix-community/nixvim/34a7d94cdcd2b034eb06202992bed1345aa046c9?narHash=sha256-fbsopU0qWfqq1WRKjWYpYCMxmEYyq%2BCmw%2B%2BVXVke5Ns%3D' (2026-01-24)
• Updated input 'rust-overlay':
    'github:oxalica/rust-overlay/03dda130a8701b08b0347fcaf850a190c53a3c1e?narHash=sha256-Tg/kL8eKMpZtceDvBDQYU8zowgpr7ucFRnpP/AtfuRM%3D' (2026-01-13)
  → 'github:oxalica/rust-overlay/22da29e7f3d8cff75009cbbcf992c7cb66920cfd?narHash=sha256-gu6oZ86zLudBZMq8LL1qdtYt/S69GV5keQVXdvBrVSU%3D' (2026-01-24)
• Updated input 'search':
    'github:NuschtOS/search/391cd4a77d6258bd9ac25c6ff822a6031610098b?narHash=sha256-XTQ%2BqroGN5XorMhynTv2u/myPRQFawjtfYxmlXjJDlo%3D' (2026-01-13)
  → 'github:NuschtOS/search/089643c6ac693c33914f4c554991935080e86018?narHash=sha256-ivDFUihujxlu1EzzzSIIEi9gM6l0kdGb76NBDYee2as%3D' (2026-01-24)
• Updated input 'search/ixx':
    'github:NuschtOS/ixx/6c74eb8eb1c78f4ef350173b77ac5145189d8e82?narHash=sha256-9XVmnC3sW/flvGCrG0d8Vo9Oe8jKOo6KgK%2B6v/xt1UI%3D' (2026-01-13)
  → 'github:NuschtOS/ixx/f1aea5f672092d621e56d55f46fedea5c3fad5a2?narHash=sha256-sMzdDUgPEV2pJbjYc%2B7Y6ThQTIdFLODWcnb9f4tL6jY%3D' (2026-01-21)
• Updated input 'sops-nix':
    'github:Mic92/sops-nix/691b8b6713855d0fe463993867291c158472fc6f?narHash=sha256-jJqlW8A3OZ5tYbXphF7U8P8g/3Cn8PPwPa4YlJ/9agg%3D' (2026-01-13)
  → 'github:Mic92/sops-nix/c7067be8db2c09ab1884de67ef6c4f693973f4a2?narHash=sha256-1IHAeS8WtBiEo5XiyJBHOXMzECD6aaIOJmpQKzRRl64%3D' (2026-01-19)

SuperSandro2000 committed:
Load flakes packages' easier

SuperSandro2000 committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'lanzaboote':
    'github:nix-community/lanzaboote/1bea6e953d06da77729edd0004291ced527bcb4a?narHash=sha256-xGlAdk2c1mVxOTMzzCYHDYuXaBMoH1BTr2nJOGkY/SQ%3D' (2026-01-23)
  → 'github:nix-community/lanzaboote/4673781e286eeeb35fec07b01cad87d41d94a349?narHash=sha256-XxYnvAPv2kZBTUywng8mvX2OJtq0Dx3FDyta1NmSjRc%3D' (2026-01-25)
• Updated input 'lanzaboote/crane':
    'github:ipetkov/crane/2fb033290bf6b23f226d4c8b32f7f7a16b043d7e?narHash=sha256-9/9ntI0D%2BHbN4G0TrK3KmHbTvwgswz7p8IEJsWyef8Q%3D' (2026-01-07)
  → 'github:ipetkov/crane/0314e365877a85c9e5758f9ea77a9972afbb4c21?narHash=sha256-gABuYA6BzoRMLuPaeO5p7SLrpd4qExgkwEmYaYQY4bM%3D' (2026-01-24)
• Updated input 'nixos-hardware':
    'github:nixos/nixos-hardware/9f7ba891ea5fc3ededd7804f1a23fafadbcb26ca?narHash=sha256-3ymIZ8s3%2Bhu7sDl/Y48o6bwMxorfKrmn97KuWiw1vjY%3D' (2026-01-22)
  → 'github:nixos/nixos-hardware/a351494b0e35fd7c0b7a1aae82f0afddf4907aa8?narHash=sha256-QEDtctEkOsbx8nlFh4yqPEOtr4tif6KTqWwJ37IM2ds%3D' (2026-01-25)
• Updated input 'search':
    'github:NuschtOS/search/089643c6ac693c33914f4c554991935080e86018?narHash=sha256-ivDFUihujxlu1EzzzSIIEi9gM6l0kdGb76NBDYee2as%3D' (2026-01-24)
  → 'github:NuschtOS/search/0534acf22fba30f290f46547f0b1c6508724788b?narHash=sha256-ZEjP6XvOOOv3VmgAzCo9hUmkMoj4fE8HDFkZPlpHaZA%3D' (2026-01-25)

SuperSandro2000 committed:
Add all available packages

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'agenix':
    'github:ryantm/agenix/fcdea223397448d35d9b31f798479227e80183f6?narHash=sha256-wyT7Pl6tMFbFrs8Lk/TlEs81N6L%2BVSybPfiIgzU8lbQ%3D' (2025-11-08)
  → 'github:ryantm/agenix/b027ee29d959fda4b60b57566d64c98a202e0feb?narHash=sha256-9VnK6Oqai65puVJ4WYtCTvlJeXxMzAp/69HhQuTdl/I%3D' (2026-02-04)
• Updated input 'authentik-nix':
    'github:nix-community/authentik-nix/1cab906a5cb342a4890ea9e4fe3993c6d438689b?narHash=sha256-9eiLAIUI3rsjqdY32%2BjQdKB%2B0VI6Jks0uf0s/UVMVJI%3D' (2026-01-24)
  → 'github:nix-community/authentik-nix/88a8771c30c0663e4789fac67fa415e7d5554715?narHash=sha256-VqF1ueagt1%2Bb1e6mrJ5%2Brs9dk3OcINxtwZgbAg6suUI%3D' (2026-05-26)
• Added input 'authentik-nix/authentik-go':
    'github:goauthentik/client-go/58f64509446aab6bc2d9b1fe36be19b5f2a3b4a8?narHash=sha256-K0dzj%2BGnajGz63Gp%2BNYyhanjhtDxpzCKpmdIZYMfz/M%3D' (2026-05-13)
• Updated input 'authentik-nix/authentik-src':
    'github:ma27/authentik/72ad5fe320f2201fc2a37372d4c9cb46377a58e5?narHash=sha256-HDTbQB/sMhYh2b95dQwzF8OgrwLWdl4hVmx6wtDcgE8%3D' (2026-01-16)
  → 'github:goauthentik/authentik/095e2897d52397962d153be2d689cedcdb71906c?narHash=sha256-tTBGwajdtEPuk36va5HgnnqR8sTuSwIt/c48bIinRlA%3D' (2026-05-12)
• Updated input 'authentik-nix/pyproject-build-systems':
    'github:pyproject-nix/build-system-pkgs/042904167604c681a090c07eb6967b4dd4dae88c?narHash=sha256-4bocaOyLa3AfiS8KrWjZQYu%2BIAta05u3gYZzZ6zXbT0%3D' (2025-11-20)
  → 'github:pyproject-nix/build-system-pkgs/7bff980f37fc24e09dbc986643719900c139bf12?narHash=sha256-MbXylBTkWqVm8/VYjoULtMoVRgWBN1gSHbeRKsOsPlU%3D' (2026-05-25)
• Updated input 'authentik-nix/pyproject-nix':
    'github:pyproject-nix/pyproject.nix/2c8df1383b32e5443c921f61224b198a2282a657?narHash=sha256-xaKvtPx6YAnA3HQVp5LwyYG1MaN4LLehpQI8xEdBvBY%3D' (2025-11-26)
  → 'github:pyproject-nix/pyproject.nix/69f57f27e52a87c54e28138a75ec741cd46663c9?narHash=sha256-Gs1VnEkCkkRZxJQAC/Dhz0Jbfi22mFXChbtNg9w/Ybg%3D' (2026-04-20)
• Updated input 'authentik-nix/uv2nix':
    'github:pyproject-nix/uv2nix/4cca323a547a1aaa9b94929c4901bed5343eafe8?narHash=sha256-90d//IZ4GXipNsngO4sb2SAPbIC/a2P%2BIAdAWOwpcOM%3D' (2025-12-13)
  → 'github:pyproject-nix/uv2nix/fdf2a76275d7a9c27deb5d2f2ab33526ac9052ff?narHash=sha256-IMFlxeyClau51KplhhSRGhdGTvD/knShHdybP1UOTuk%3D' (2026-05-22)
• Updated input 'comin':
    'github:nlewo/comin/a35bc15dacef048d3dbe611c91acf7a4c274f24e?narHash=sha256-41Ts3IL/6zD%2BenoQh3/8FCucCJn3HBsI8FbdNDG2MbU%3D' (2026-01-22)
  → 'github:nlewo/comin/cfe05b79cb31a635e3410b514bd5503642e16117?narHash=sha256-Wevj/TCya0rhH3%2ByuD6RwwPGi2PBBZADP3bF8/9f3Js%3D' (2026-05-23)
• Added input 'comin/treefmt-nix':
    'github:numtide/treefmt-nix/337a4fe074be1042a35086f15481d763b8ddc0e7?narHash=sha256-wQ6NJSuFqAEmIg2VMnLdCnUc0b7vslUohqqGGD%2BFyxk%3D' (2026-02-04)
• Added input 'comin/treefmt-nix/nixpkgs':
    'github:nixos/nixpkgs/4533d9293756b63904b7238acb84ac8fe4c8c2c4?narHash=sha256-tbS0Ebx2PiA1FRW8mt8oejR0qMXmziJmPaU1d4kYY9g%3D' (2026-02-03)
• Updated input 'disko':
    'github:nix-community/disko/00395d188e3594a1507f214a2f15d4ce5c07cb28?narHash=sha256-GVJ0jKsyXLuBzRMXCDY6D5J8wVdwP1DuQmmvYL/Vw/Q%3D' (2026-01-20)
  → 'github:nix-community/disko/5ba0c9555c28685e57fa54c7a25e42c7efdbfc8d?narHash=sha256-EcCaSTKnmg2o4wLKaN1aqQFomwyhO7ik0bX9COdyCas%3D' (2026-05-25)
• Updated input 'dn42-nix':
    'github:NuschtOS/dn42.nix/307373d46f0dbead001426907de710b3f3c63eec?narHash=sha256-lCkx2EKFGKJGxIQ%2BGVqX/m99tE7HT5ItIrhE%2BhDLlHM%3D' (2026-01-04)
  → 'github:NuschtOS/dn42.nix/9dd409947b02b99c11f0e1a9f2f829813ad432c5?narHash=sha256-sKC%2BiM0LFa7TPhU18DxkCg0aw1roZ75jOfJ1u7upyVc%3D' (2026-05-02)
• Updated input 'flake-parts':
    'github:hercules-ci/flake-parts/80daad04eddbbf5a4d883996a73f3f542fa437ac?narHash=sha256-PVvu7OqHBGWN16zSi6tEmPwwHQ4rLPU9Plvs8/1TUBY%3D' (2026-01-11)
  → 'github:hercules-ci/flake-parts/f7c1a2d347e4c52d5fb8d10cb4d94b5884e546fb?narHash=sha256-m1Yf0wZ8j1OHjTc2UwHwyQRSnNeSgLJOd7q5Y45hzi4%3D' (2026-05-13)
• Updated input 'home-manager':
    'github:nix-community/home-manager/2539eba97a6df237d75617c25cd2dbef92df3d5b?narHash=sha256-6Cwtvzrw79cOk1lCzN2aKSVrpgSOSQoYhyMmhXXZjTA%3D' (2026-01-24)
  → 'github:nix-community/home-manager/1a95e2efb477959b70b4a14c51035975c0481df6?narHash=sha256-/p37CB5n6Wpw250b0Lq0CYwNq2D8uGKzDoBulyLcQqA%3D' (2026-05-25)
• Updated input 'impermanence':
    'github:nix-community/impermanence/69ecf31e8fddc9354a4b418f3a517445d486bb54?narHash=sha256-OyxsfXNcOkt06/kM%2B4bnuC8moDx%2Bt7Qr%2BRB0BBa83Ig%3D' (2026-01-20)
  → 'github:nix-community/impermanence/7b1d382faf603b6d264f58627330f9faa5cba149?narHash=sha256-03%2BJxvzmfwRu%2B5JafM0DLbxgHttOQZkUtDWBmeUkN8Y%3D' (2026-01-27)
• Updated input 'lanzaboote':
    'github:nix-community/lanzaboote/4673781e286eeeb35fec07b01cad87d41d94a349?narHash=sha256-XxYnvAPv2kZBTUywng8mvX2OJtq0Dx3FDyta1NmSjRc%3D' (2026-01-25)
  → 'github:nix-community/lanzaboote/b7e492b90bcae5e9d9cb0a1d4d1e6100305de2c5?narHash=sha256-FHKdAxS5TJ0mdNS/9YuIam7NGtzEXVggLEWQ00Q31yc%3D' (2026-05-18)
• Updated input 'lanzaboote/crane':
    'github:ipetkov/crane/0314e365877a85c9e5758f9ea77a9972afbb4c21?narHash=sha256-gABuYA6BzoRMLuPaeO5p7SLrpd4qExgkwEmYaYQY4bM%3D' (2026-01-24)
  → 'github:ipetkov/crane/10e6e3cb966f7cfcc789fe5eee7a85f3188ce08b?narHash=sha256-nnGD2f8OlAZT2i5OfwikJsw%2BifWfiA4d6A8BWlgOXV0%3D' (2026-05-17)
• Updated input 'microvm':
    'github:microvm-nix/microvm.nix/f469c1dfede623bbbf1ac605f6359316fd4002ef?narHash=sha256-mKrMf7eG9TM2AM3pTuhIiCGmZ/JwDegCQH3ThVqcTuc%3D' (2026-01-17)
  → 'github:microvm-nix/microvm.nix/4fe5ab55ef830e4b2d82d37ef7a5aca6e7b0e5f8?narHash=sha256-xSEiY1kZoCHX7bzeDe6fzwnaG9Mjd/WrJ4MEWJghI/Q%3D' (2026-05-24)
• Updated input 'microvm/spectrum':
    'git+https://spectrum-os.org/git/spectrum?ref=refs/heads/main&rev=c5d5786d3dc938af0b279c542d1e43bce381b4b9' (2025-10-03)
  → 'git+https://spectrum-os.org/git/spectrum?ref=refs/heads/main&rev=367dd227f539267eae2b62770b4c17b88ac8c1f1' (2026-05-16)
• Updated input 'nix-darwin':
    'github:nix-darwin/nix-darwin/0fc4e7ac670a0ed874abacf73c4b072a6a58064b?narHash=sha256-5ulSDyOG1U%2B1sJhkJHYsUOWEsmtLl97O0NTVMvgIVyc%3D' (2026-01-18)
  → 'github:nix-darwin/nix-darwin/56c666e108467d87d13508936aade6d567f2a501?narHash=sha256-zXcwYQGCT6pzinK%2B1dBB2ekTVtfxGZAapb3Evdcu4fY%3D' (2026-05-17)
• Updated input 'nixos-apple-silicon':
    'github:tpwrules/nixos-apple-silicon/003afbd7b4f90c83497906ba2e9099874c2e3a4f?narHash=sha256-OA%2BQrrf0FWqARjgJASGrPKMUw237G3gYG%2BMQn4upABU%3D' (2026-01-23)
  → 'github:tpwrules/nixos-apple-silicon/f96c6b94d4261526a3911f28aea7ca5033bcae0a?narHash=sha256-stdabStuiiwzrNMXMr8ZWZXPOlEIiss9AUEpYF2vavk%3D' (2026-05-25)
• Updated input 'nixos-hardware':
    'github:nixos/nixos-hardware/a351494b0e35fd7c0b7a1aae82f0afddf4907aa8?narHash=sha256-QEDtctEkOsbx8nlFh4yqPEOtr4tif6KTqWwJ37IM2ds%3D' (2026-01-25)
  → 'github:nixos/nixos-hardware/c97bc4d15bd3473dd095e8e8ba57330ab1943a77?narHash=sha256-j1iZsLy6oFApqR1oiDmHhvkwxXqcNi0aoSJj643LuwU%3D' (2026-05-20)
• Updated input 'nixos-modules':
    'github:NuschtOS/nixos-modules/31108e0d75bd47ddfc217b58df598e78fe3bcd42?narHash=sha256-hmIo/e6mo40Y2v1DaH2yTtvB3lZ/zcf6gVNmgYhBgYc%3D' (2026-01-15)
  → 'github:NuschtOS/nixos-modules/750fa3cbfc7d3c643afa28a6a56b565b4d3d4f4d?narHash=sha256-W0trjXe5AWi0jhPq6g8FG%2BZ5QYEMYxUDpMenFk8S%2BDY%3D' (2026-05-21)
• Updated input 'nixos-wsl':
    'github:nix-community/NixOS-WSL/38a5250e57f583662eac3b944830e4b9e169e965?narHash=sha256-RY9kJDXD6%2B2Td/59LkZ0PFSereCXHdBX9wIkbYjRKCY%3D' (2026-01-24)
  → 'github:nix-community/NixOS-WSL/5482f113fd31ebac131d1ebeb2ae90bf0d5e41f5?narHash=sha256-2uX/XtOWZ/oy2rerRynVhqVA//ZXZ3Fo60PikLHEPQc%3D' (2026-05-02)
• Updated input 'nixpkgs':
    'github:nuschtos/nuschtpkgs/347928902d4292ce08651b97c1264edf7a342db1?narHash=sha256-VyRjKmCv3iCDjSkDygD0YT97IB0xRQJO2qlsC/B6QSw%3D' (2026-01-22)
  → 'github:nuschtos/nuschtpkgs/cc8d3defe8697c5325733952fdac1740d5d420a4?narHash=sha256-2aNlBiC63HeCxGzCaYU0UQ1CcKeNNwll9CoRAvw7Apk%3D' (2026-05-25)
• Updated input 'nixvim':
    'github:nix-community/nixvim/34a7d94cdcd2b034eb06202992bed1345aa046c9?narHash=sha256-fbsopU0qWfqq1WRKjWYpYCMxmEYyq%2BCmw%2B%2BVXVke5Ns%3D' (2026-01-24)
  → 'github:nix-community/nixvim/297f9341476ba7f821a42d7a2805e206ef8c6ef8?narHash=sha256-Kgod3gZlhSp6WozZ2pFaclXbWpjs6kQLAtldoxb85Lc%3D' (2026-05-26)
• Updated input 'rust-overlay':
    'github:oxalica/rust-overlay/22da29e7f3d8cff75009cbbcf992c7cb66920cfd?narHash=sha256-gu6oZ86zLudBZMq8LL1qdtYt/S69GV5keQVXdvBrVSU%3D' (2026-01-24)
  → 'github:oxalica/rust-overlay/09889992e640378f7008c3302b9d4892579df610?narHash=sha256-2QxuD5gJcfCFsnqv54LGEHQKvd%2BK%2BDW/ecERwAvuA7w%3D' (2026-05-26)
• Updated input 'search':
    'github:NuschtOS/search/0534acf22fba30f290f46547f0b1c6508724788b?narHash=sha256-ZEjP6XvOOOv3VmgAzCo9hUmkMoj4fE8HDFkZPlpHaZA%3D' (2026-01-25)
  → 'github:NuschtOS/search/c96541f53ff5e97a33da4f434a1df73df3a40719?narHash=sha256-s6vdZEex1dSrT5HJFgPpPo0MQCzUJQG0gX%2Bso4R3eeM%3D' (2026-05-26)
• Updated input 'search/ixx':
    'github:NuschtOS/ixx/f1aea5f672092d621e56d55f46fedea5c3fad5a2?narHash=sha256-sMzdDUgPEV2pJbjYc%2B7Y6ThQTIdFLODWcnb9f4tL6jY%3D' (2026-01-21)
  → 'github:NuschtOS/ixx/7aeebd04a4886755e2b24b2510280628b7a0ddb4?narHash=sha256-ol%2BqY5b6AE2TKsFsLheowFAhv94H8234MGZToNc2XOQ%3D' (2026-03-17)
• Updated input 'search/nix-index-database':
    'github:Mic92/nix-index-database/82befcf7dc77c909b0f2a09f5da910ec95c5b78f?narHash=sha256-d3NBA9zEtBu2JFMnTBqWj7Tmi7R5OikoU2ycrdhQEws%3D' (2025-12-09)
  → 'github:Mic92/nix-index-database/8fba98c80b48fa013820e0163c5096922fea4ddd?narHash=sha256-ZQ5z%2BfVhxYKtIFwtqGp5O0PD84BM1riASvqDaN5Xs%2Bs%3D' (2026-05-24)
• Updated input 'simple-nixos-mailserver':
    'git+https://gitlab.com/simple-nixos-mailserver/nixos-mailserver.git?ref=refs/heads/master&rev=7d433bf89882f61621f95082e90a4ab91eb0bdd3' (2025-12-21)
  → 'git+https://gitlab.com/simple-nixos-mailserver/nixos-mailserver.git?ref=refs/heads/main&rev=da16ebd4b5f8c2867f0a5220780f87d94d979626' (2026-05-24)
• Updated input 'simple-nixos-mailserver/git-hooks':
    'github:cachix/git-hooks.nix/50b9238891e388c9fdc6a5c49e49c42533a1b5ce?narHash=sha256-QlcnByMc8KBjpU37rbq5iP7Cp97HvjRP0ucfdh%2BM4Qc%3D' (2025-11-24)
  → 'github:cachix/git-hooks.nix/61ab0e80d9c7ab14c256b5b453d8b3fb0189ba0a?narHash=sha256-kTwur1wV%2B01SdqskVMSo6JMEpg71ps3HpbFY2GsflKs%3D' (2026-05-11)
• Updated input 'sops-nix':
    'github:Mic92/sops-nix/c7067be8db2c09ab1884de67ef6c4f693973f4a2?narHash=sha256-1IHAeS8WtBiEo5XiyJBHOXMzECD6aaIOJmpQKzRRl64%3D' (2026-01-19)
  → 'github:Mic92/sops-nix/c591bf665727040c6cc5cb409079acb22dcce33c?narHash=sha256-VfGRo1qTBKOe3s2gOv8LSoA6Fk19PvBlwQ1ECN0Evn8%3D' (2026-05-05)
• Updated input 'tsnsrv':
    'github:boinkor-net/tsnsrv/8f3fbf69e1517612c0fde9be27522e12c2ddc238?narHash=sha256-uRm4oWMFUBSmXgofOngMWIyA/yUmLlon6f3XoFTWvBA%3D' (2025-12-29)
  → 'github:boinkor-net/tsnsrv/9f36a4f264540d05b8f758a67eee3e57dca4eea6?narHash=sha256-DqieJ48Jf%2BgmJtfMnE7QXpnJnW7acnT624N/iA6pzVI%3D' (2026-03-17)

MarcelCoding committed:
flake: fix build

MarcelCoding committed:
Merge remote-tracking branch 'origin/main' into packages

MarcelCoding committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'authentik-nix':
    'github:nix-community/authentik-nix/88a8771c30c0663e4789fac67fa415e7d5554715?narHash=sha256-VqF1ueagt1%2Bb1e6mrJ5%2Brs9dk3OcINxtwZgbAg6suUI%3D' (2026-05-26)
  → 'github:nix-community/authentik-nix/7611d20f7a0db3e08bd94b3cb3948d8e38958776?narHash=sha256-zAJcOvOHBdaqnmGXxeXPwIgQ3p4VDMt8ecJ2sl4cdjQ%3D' (2026-06-04)
• Removed input 'authentik-nix/authentik-go'
• Updated input 'authentik-nix/authentik-src':
    'github:goauthentik/authentik/095e2897d52397962d153be2d689cedcdb71906c?narHash=sha256-tTBGwajdtEPuk36va5HgnnqR8sTuSwIt/c48bIinRlA%3D' (2026-05-12)
  → 'github:goauthentik/authentik/2d26ce9b538df68b9d78dccdd5ac5bfb4c1b2983?narHash=sha256-djsdph2E%2BJIIu9Sy5gJwBG378nunXXK2LPW5z/oneMs%3D' (2026-05-28)
• Updated input 'authentik-nix/pyproject-nix':
    'github:pyproject-nix/pyproject.nix/69f57f27e52a87c54e28138a75ec741cd46663c9?narHash=sha256-Gs1VnEkCkkRZxJQAC/Dhz0Jbfi22mFXChbtNg9w/Ybg%3D' (2026-04-20)
  → 'github:pyproject-nix/pyproject.nix/a228447c3e179d477c1b6246ef3efa8cfe3c469a?narHash=sha256-GSKXTAnFqRAMlZkJrIPcQMYf%2BlpMr66K3i60mB9STvc%3D' (2026-05-16)
• Updated input 'bird-nix':
    'github:NuschtOS/bird.nix/add53b9595e2e910ba58d552f2c42004babbb056?narHash=sha256-XsHjComfnibSuQtV/mfOVpLijXElPgt6e/2fVL1NAbs%3D' (2026-01-17)
  → 'github:NuschtOS/bird.nix/fdcf7b7de4e91329882b906f5389eed2f64a7f48?narHash=sha256-SS5vWdSceFObWvfJHBWXGF63gaTIjWCP%2BRVMb33zRbM%3D' (2026-06-05)
• Updated input 'comin':
    'github:nlewo/comin/cfe05b79cb31a635e3410b514bd5503642e16117?narHash=sha256-Wevj/TCya0rhH3%2ByuD6RwwPGi2PBBZADP3bF8/9f3Js%3D' (2026-05-23)
  → 'github:nlewo/comin/56320254c8e651c052fb1a175470821a476c5bb1?narHash=sha256-Ad%2BS86WqmtQWGSg0KgSjMwMXq%2Bya0HDeF4m89Kn463o%3D' (2026-06-01)
• Updated input 'comin/treefmt-nix/nixpkgs':
    'github:nixos/nixpkgs/4533d9293756b63904b7238acb84ac8fe4c8c2c4?narHash=sha256-tbS0Ebx2PiA1FRW8mt8oejR0qMXmziJmPaU1d4kYY9g%3D' (2026-02-03)
  → follows 'nixpkgs'
• Updated input 'disko':
    'github:nix-community/disko/5ba0c9555c28685e57fa54c7a25e42c7efdbfc8d?narHash=sha256-EcCaSTKnmg2o4wLKaN1aqQFomwyhO7ik0bX9COdyCas%3D' (2026-05-25)
  → 'github:nix-community/disko/115e5211780054d8a890b41f0b7734cafad54dfe?narHash=sha256-eTAlX0CwgB84Ts3GaBd944A3DRXVMzgA0EqroZBISUo%3D' (2026-06-01)
• Updated input 'dn42-nix':
    'github:NuschtOS/dn42.nix/9dd409947b02b99c11f0e1a9f2f829813ad432c5?narHash=sha256-sKC%2BiM0LFa7TPhU18DxkCg0aw1roZ75jOfJ1u7upyVc%3D' (2026-05-02)
  → 'github:NuschtOS/dn42.nix/86a9676459dfe9e3d51f61865211ae8674dfc106?narHash=sha256-4/dP6aNkEoAnClhn9Djk1JbXNfgfnPTcZt7H%2BZBox%2BQ%3D' (2026-06-05)
• Updated input 'home-manager':
    'github:nix-community/home-manager/1a95e2efb477959b70b4a14c51035975c0481df6?narHash=sha256-/p37CB5n6Wpw250b0Lq0CYwNq2D8uGKzDoBulyLcQqA%3D' (2026-05-25)
  → 'github:nix-community/home-manager/b2b7db486e06e098711dc291bb25db82850e1d16?narHash=sha256-KmRNvpNOb7QEORa06bVgjW9kITcx0VhsI7w0vhmZyD8%3D' (2026-06-05)
• Updated input 'lanzaboote':
    'github:nix-community/lanzaboote/b7e492b90bcae5e9d9cb0a1d4d1e6100305de2c5?narHash=sha256-FHKdAxS5TJ0mdNS/9YuIam7NGtzEXVggLEWQ00Q31yc%3D' (2026-05-18)
  → 'github:nix-community/lanzaboote/e780b8e19d405b6906d759d270bbc125d221e81a?narHash=sha256-2YH74Fj5%2B/gc94qW7ZMN0YF3sMhf2BFB5NBaJgDsugo%3D' (2026-06-04)
• Updated input 'lanzaboote/crane':
    'github:ipetkov/crane/10e6e3cb966f7cfcc789fe5eee7a85f3188ce08b?narHash=sha256-nnGD2f8OlAZT2i5OfwikJsw%2BifWfiA4d6A8BWlgOXV0%3D' (2026-05-17)
  → 'github:ipetkov/crane/0532eb17955225173906d671fb36306bdeb1e2dc?narHash=sha256-EVZd2RsbpreRUDSi9rBwPY%2BZxoyMaiEBbZxxhljbaS4%3D' (2026-05-30)
• Updated input 'microvm':
    'github:microvm-nix/microvm.nix/4fe5ab55ef830e4b2d82d37ef7a5aca6e7b0e5f8?narHash=sha256-xSEiY1kZoCHX7bzeDe6fzwnaG9Mjd/WrJ4MEWJghI/Q%3D' (2026-05-24)
  → 'github:microvm-nix/microvm.nix/4d3fb17437944ea57eef2b9e6108ca777b1209ca?narHash=sha256-zQk%2BGqLO%2BT9taIl1UUt3swvaOksWJxL7PL8K0%2BFc/Hs%3D' (2026-06-04)
• Updated input 'nix-darwin':
    'github:nix-darwin/nix-darwin/56c666e108467d87d13508936aade6d567f2a501?narHash=sha256-zXcwYQGCT6pzinK%2B1dBB2ekTVtfxGZAapb3Evdcu4fY%3D' (2026-05-17)
  → 'github:nix-darwin/nix-darwin/6a771120d607dcccb279a27d227650e324815c35?narHash=sha256-AkWx4Zt9pQbD/f82Z8N57%2Bd0HGLN/rV3gdMKJTpBPKs%3D' (2026-06-07)
• Updated input 'nixos-apple-silicon':
    'github:tpwrules/nixos-apple-silicon/f96c6b94d4261526a3911f28aea7ca5033bcae0a?narHash=sha256-stdabStuiiwzrNMXMr8ZWZXPOlEIiss9AUEpYF2vavk%3D' (2026-05-25)
  → 'github:tpwrules/nixos-apple-silicon/5880026520a3fd248d59e1c81c4e4e111aefc6af?narHash=sha256-inOQx/s7GQjh9bcCjCHXAeX0EHX%2BsOQUBoo8%2Bbs48ME%3D' (2026-06-05)
• Updated input 'nixos-hardware':
    'github:nixos/nixos-hardware/c97bc4d15bd3473dd095e8e8ba57330ab1943a77?narHash=sha256-j1iZsLy6oFApqR1oiDmHhvkwxXqcNi0aoSJj643LuwU%3D' (2026-05-20)
  → 'github:nixos/nixos-hardware/4ed851c979641e28597a05086332d75cdc9e395f?narHash=sha256-fPBRVf6A5xlACYcOI59shGrjURuvwu0lRsDoSCEXt/I%3D' (2026-06-01)
• Added input 'nixos-hardware/nixpkgs':
    'https://releases.nixos.org/nixos/unstable/nixos-26.05pre924538.3497aa5c9457/nixexprs.tar.xz?narHash=sha256-8bW3q88CEg2u4hSP66Vf4lpbLonHz7hqDNBMcCY7E9U%3D' (2026-01-08)
• Updated input 'nixos-modules':
    'github:NuschtOS/nixos-modules/750fa3cbfc7d3c643afa28a6a56b565b4d3d4f4d?narHash=sha256-W0trjXe5AWi0jhPq6g8FG%2BZ5QYEMYxUDpMenFk8S%2BDY%3D' (2026-05-21)
  → 'github:NuschtOS/nixos-modules/32c074f6c3bcb1ed0f7ff7ced4acf99b51abd07a?narHash=sha256-mhxSFXG/Pm4RnsOaj9j92R2GjGzlBLmEQ/bn4Yx73GU%3D' (2026-06-05)
• Updated input 'nixos-wsl':
    'github:nix-community/NixOS-WSL/5482f113fd31ebac131d1ebeb2ae90bf0d5e41f5?narHash=sha256-2uX/XtOWZ/oy2rerRynVhqVA//ZXZ3Fo60PikLHEPQc%3D' (2026-05-02)
  → 'github:nix-community/NixOS-WSL/3e6d8af994e2a2d31af7a91863d7c0d6e278d951?narHash=sha256-md6QHmlIx40bQkun43M2eT8aav5GURGkXEMFwof6uZs%3D' (2026-06-06)
• Updated input 'nixpkgs':
    'github:nuschtos/nuschtpkgs/cc8d3defe8697c5325733952fdac1740d5d420a4?narHash=sha256-2aNlBiC63HeCxGzCaYU0UQ1CcKeNNwll9CoRAvw7Apk%3D' (2026-05-25)
  → 'github:nuschtos/nuschtpkgs/331800de5053fcebacf6813adb5db9c9dca22a0c?narHash=sha256-x5UQuRsH3MqI0U9afaXSNqzTPSeZlRLvFAav2Ux1pNw%3D' (2026-05-31)
• Updated input 'nixvim':
    'github:nix-community/nixvim/297f9341476ba7f821a42d7a2805e206ef8c6ef8?narHash=sha256-Kgod3gZlhSp6WozZ2pFaclXbWpjs6kQLAtldoxb85Lc%3D' (2026-05-26)
  → 'github:nix-community/nixvim/98acb923e5882a6a6a2b660d99e8c5124e08b838?narHash=sha256-yjtMubbmPi4Y4pgDSdmytqUutGJ%2BLJLcPgXGNNdXiUs%3D' (2026-06-07)
• Updated input 'rust-overlay':
    'github:oxalica/rust-overlay/09889992e640378f7008c3302b9d4892579df610?narHash=sha256-2QxuD5gJcfCFsnqv54LGEHQKvd%2BK%2BDW/ecERwAvuA7w%3D' (2026-05-26)
  → 'github:oxalica/rust-overlay/8e596a8430f2ce54d55c742198187d6945a5501e?narHash=sha256-bGtIUeLb0yChX4h6hB40OOCwcYhcpQZHXSDvZGdWgeM%3D' (2026-06-07)
• Updated input 'search':
    'github:NuschtOS/search/c96541f53ff5e97a33da4f434a1df73df3a40719?narHash=sha256-s6vdZEex1dSrT5HJFgPpPo0MQCzUJQG0gX%2Bso4R3eeM%3D' (2026-05-26)
  → 'github:NuschtOS/search/9159ac4277099ad3c6f5ee34c5f928cb0b7f436b?narHash=sha256-jjfsPLKftyY5t7mVD6yT5FtI%2BxrGcFx6ASdnBDqlqYI%3D' (2026-06-07)
• Updated input 'search/ixx':
    'github:NuschtOS/ixx/7aeebd04a4886755e2b24b2510280628b7a0ddb4?narHash=sha256-ol%2BqY5b6AE2TKsFsLheowFAhv94H8234MGZToNc2XOQ%3D' (2026-03-17)
  → 'github:NuschtOS/ixx/d7a538df37375683fda931d244ca47aed0296940?narHash=sha256-vLSO1r6UJ8%2B8r2o05ZILtvIQ5vPobGWTKvWXc4hYq%2Bo%3D' (2026-06-07)
• Updated input 'search/nix-index-database':
    'github:Mic92/nix-index-database/8fba98c80b48fa013820e0163c5096922fea4ddd?narHash=sha256-ZQ5z%2BfVhxYKtIFwtqGp5O0PD84BM1riASvqDaN5Xs%2Bs%3D' (2026-05-24)
  → 'github:Mic92/nix-index-database/1a2ea89c917781e88508d9fd2b507f2d2a0e173c?narHash=sha256-0BYqs8yKWkOz2Q7%2BSP18N5E5gmDKSo6LSxIVIa0wWes%3D' (2026-06-07)
• Updated input 'simple-nixos-mailserver':
    'git+https://gitlab.com/simple-nixos-mailserver/nixos-mailserver.git?ref=refs/heads/main&rev=da16ebd4b5f8c2867f0a5220780f87d94d979626' (2026-05-24)
  → 'git+https://gitlab.com/simple-nixos-mailserver/nixos-mailserver.git?ref=refs/heads/main&rev=124cd33453267157bf3cb15b66bd9be99b8d1067' (2026-06-05)
• Updated input 'sops-nix':
    'github:Mic92/sops-nix/c591bf665727040c6cc5cb409079acb22dcce33c?narHash=sha256-VfGRo1qTBKOe3s2gOv8LSoA6Fk19PvBlwQ1ECN0Evn8%3D' (2026-05-05)
  → 'github:Mic92/sops-nix/9ed65852b6257fbeae4355bc24ecfea307ca759a?narHash=sha256-Gq8KNx5A7hBB3uGJaj6eQfLDIz5YdLu92gqBcvHvoUo%3D' (2026-06-04)

MarcelCoding committed:
Merge remote-tracking branch 'origin/main' into packages
